### DISCUSSION TITLE
High Gas Fees - Block Bucks

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#2201] [Ideas]
_The following is in draft mode - it is an early idea but we wanted to gather community feedback since this is a pressing concern. We will share this in announcements once details are finalised_

The purpose of this proposal is to discuss the issue of high gas fees that have been occurring across Polygon the last 5 days. Recently these fees have been 5x due to a range of new projects on the Blockchain.

Unfortunately this has led to players unable to sync, expensive fees and/or failed transactions.

The team are investigating an approach that will ensure gameplay is not drastically affected during these periods of high traffic on the Blockchain.

**The balance of Web2 and Web3**

The Sunflower Land team is deeply passionate about on-chain gameplay, evident in our prototype which was 100% on the Blockchain. Since then we have learnt it is best to perform some actions on a game server and reserve others for Blockchain. This reduces transaction fees and improves the games UX significantly without the constant need to wait for transactions.

There is a fine balance that we strive for when integrating Web2 and Web3 elements. Our goal continues to maximise on chain activity whilst ensuring the game is playable and user friendly.

## Block Bucks

We are interested in launching the idea of "Block Bucks" - name is to be decided, please provide suggestions 🙏 

1 Block Buck could be used to **instantly** restock your shops, instead of 'Syncing your data on chain'. This means players would not need to wait for a transaction and pay the associated gas fees.

**How do you get Block Bucks?**

You would be able to purchase 5 Block Bucks within the game for **$X**. This would trigger a transaction that would:

1. Sync your data on Chain
2. Add **X** Block Buck tickets to your inventory.

Once the transaction completes, you can then use these tickets to restock your inventory.

The approach above enables a player to restock their shop 5 times whilst only paying the gas fees on 1 transaction. In essence, a player pays roughly 1/5th of the gas fees.

**Pros**

- Players spend less time waiting for transactions / restocks are instant
- Players spend less money on gas fees

**Cons**

- On chain daily active users would decrease
- On chain data is available later than normal (only once a user gets Block Bucks)
- Input price for resources decreases. If players are paying less gas fees, MATIC input is less for output of resources

On a side note, the team will be investigating a Daily Rewards on chain transaction that can use insignificant gas (<0.02) - this will become the main driver for capturing DAU (Daily Active Users)

## Pricing Structure

We want to encourage users to sync on chain more often to ensure we remain that balance between web2 and web3 gaming. The pricing structure we propose takes an inverse structure to buying in bulk. There are multiple options to choose which enables players to adapt to current blockchain conditions.

The following calculations assume:

- Average gas prices for a sync are around $0.11-0.13
- Spiked gas prices for a sync current are around $0.30-$0.35

1 Buck = $0.10 (current price in game)
5 Bucks = $0.75 Total =  $0.15 each

When a Block Buck is purchased, the standard gas fees would be applied while syncing on chain. Below we look at the overall transaction price when purchasing the following with average gas fees of **$0.12**

1 Buck - $0.10 + $0.12 = $0.22 per sync (average price players were paying)
5 Bucks -($0.75 + $0.12) / 5 = $0.174 per sync

In the above scenario of average Blockchain conditions, a player who chose 5 Bucks would save roughly $0.05 per sync and an **overall savings of $0.47**

Below we look at the overall transaction price when purchasing the following with average gas fees of **$0.35**

1 Buck - $0.10 + $0.35 = $0.45 per sync (current price players are paying with spiked fees)
5 Bucks -($0.75 + $0.35) / 5 = $0.22 per sync

In the above **spiked** scenario, players who chose 5 Bucks would save roughly $0.23 per sync and an **overall savings of $1.40**

## Timeline

We estimate this feature would take roughly 2-3 days to roll out and would not have a significant impact on our roadmap items 🚀 

### Summary

Overall, the proposed Blocks solution saves money for players on gas fees in all Blockchain traffic conditions & ensures smoother of gameplay without waiting for long-running transactions.

However, we must tread carefully with this approach as it means Blockchain data becomes less 'fresh' and our perceived on chain users would decrease.

As always we appreciate all kinds of feedback and suggestions ❤️

**Comment by DemonDG** [comment]
I think this is a very good idea, and players won't forget that it's a web3 game because they save gas.

**Reply by rasheed19871** [reply]
that will be very nice i still will sync on chain but its a relief to sync twice a day just for axes and pickaxes, love you team

**Comment by SarielWang93** [comment]
Hope this will be live soon, can't wait any longer.

**Comment by DemonDG** [comment]
I just have a question, as shown in the table above
1 Buck = $0.10 (current price in game)
5 Bucks = $0.75 Total = $0.15 each
Why not 5 Bucks cost $0.5 in total?

**Reply by SarielWang93** [reply]
I think 5 Bucks = $0.75 can prevent players from abusing .Try to keep the chain 'fresh'.When the gas is small enough, it is cost-effective to choose 1 Buck.

**Reply by adamhannigan** [reply]
Yes, as @SarielWang93 mentioned 👍

**Reply by yoeldtrujillo** [reply]
Can i ask what abuse are you preventing putting the 5 bucks equal to 0.75. In case of any abuse is allowed i think 0.25 extra is not going to prevent it.  I would love to understand the reason of why 5 bucks it's not $.50 (according to the currently sync price fee).

**Comment by dragonhunter2017** [comment]
It is good idea to keep the both option (single or bulk sync). When gas is low, we can sync normal as we do now and when gas is high, we can choose the bulk option. 
The proposal of different sync fee is also reasonable. Both methods are cost effective for players.
Please implement is as soon as possible.

**Reply by lipel7** [reply]
eu concordo com isso, poderia ter as duas funções, cabe ao jogador escolher, mas gostei demais da proposta!

**Comment by C0m4nd4n73** [comment]
I think the rate for 5 bucks could be higher and it would still be worth paying it. As I undertood, having 5 bucks don't restrics syncs for 1 buck when it is cheaper. It is an opportunity to increase the project's resources and implement some improvements, in particular some quality process in order to minimize implementation errors in items, events and game modes and also strengthen the game's infrastructure, since it has added many elements offchain.

**Reply by cnmozzie** [reply]
totally agree, the team can get more matic while the player can save gas fee, win-win

**Reply by adamhannigan** [reply]
Thanks for your feedback. It is a delicate subject raising more funds for the project - right now we feel that $0.15 is a reasonable option to provide users ❤️

**Comment by Aquila199** [comment]
i like this 

i think it really important to keep both option single sync or bulk available all the times giving the player the option to pick whats best for them

**Reply by adamhannigan** [reply]
Definitely - this was part of our reasoning. We keep the 1 Block Buck for players who enjoy the current system  😄

**Reply by lipel7** [reply]
penso o mesmo, acho necessário ter duas opções de escolha!
gostei da proposta, seria bom se implementassem logo, e também deixasse a atual!

**Comment by AxieFan1** [comment]
Pleased to see that the team has taken note of the exorbitant gas fees issue on PolyGon network.
One suggestion is that just saving on 4 transactional gas fees is a little less, and thus there should be an option for 10 Block Bucks, so that we avoid more of such exorbitant gas fees. The projects that have led to the spike, are not going away anytime soon, and thus we need the option for 10 Block Bucks to avoid the steep gas fees more frequently.

**Reply by adamhannigan** [reply]
We will first experiement with 5 Bucks to assess the effects it has on our gameplay.

We will be open to adding more but want to be cautious initially 👍

**Comment by Prad0visk** [comment]
i think i didn't understand very well, someone correct or explain to me if what i say is wrong or right

we were paying around 0,20-22 matic to sync, 0,11 sunflower team and the rest as gas fees?

and this proposal is to pay 0,75 for sunflower( for 5 syncs) plus the matic gás fees?

**Reply by dragonhunter2017** [reply]
The price proposal is in USD, not in matic. Game fee is 0.1usd (0.08 matic) and gas fee is now 0.73usd (0.55 matic). 
1 time sync cost 0.83usd (0.63 matic). 

**so 5 sync cost is 4.12usd (3.15 matic)**

during bulk sync, game fee will increase to 0.15usd (around 0.12 matic).
bulk sync (5 restock) cost game fee 5x0.15usd=0.75usd (0.6 matic), gas fee is now 0.73usd (0.55 matic).

**so bulk sync cost is 1.49usd (1.15 matic)** 

so u save 2.5 usd (2 matic) in bulk sync, even though the game fee is higher.

when gas fee is normal, you will do regular 1 sync (game fee 0.1usd, 0,08 matic)

![image](https://user-images.githubusercontent.com/27850159/217557401-b54cf3ed-e7ab-47ce-a846-3e867c535c2d.png)

**Comment by rv-02** [comment]
As this has become the main concern over literally everything for most players, would be great if the team can prfioritize this, perhaps no need to overanalyse the impact and sustainability, we can always consider to make it a temporary fix, until gas is back to normal. If that makes sense on the workload wise of course.

**Comment by IvanSkazkin** [comment]
Looking forward to this feature

**Comment by ExtremeCheddar** [comment]
Is there a maximum amount of block bucks you would be able to hold? for instance when gas fees are low would i be able to purchase 10x 5 Block bucks (so 50 block bucks in total), so would save even more gas fees for when its high again.

**Reply by adamhannigan** [reply]
We are open to allowing players to purchase more Block Bucks, but right now we are hesitant as we are not certain what the outcome of this feature will be & the knock on effects

Our plan is to be cautious in our launch, limit it to 5 and then increase it if all seems well 👍

**Reply by GuilhermeSenaRJ** [reply]
I think this number is great, including an option to buy 5 at once that only pays for gas

**Comment by brian-man** [comment]
### Sales Pitch

Is the team open to "infinity supply" (infinite restocks of seeds/tools) for a monthly fee of $50 _as a new option_?
The docs still say targeted sync rate is 1-2 days, so that's much more than that rate for team income.

### Some Numbers

Let's break that down.

$50 is the equivalent (for the dev team) of 500 syncs.  In a month that's 15+ syncs per day.  So the value proposition would be that the dev team would get a subscription-style income of $50/mo from the player and the player would only benefit in game-play if they used 16 syncs worth of seeds and/or tools per day.

### Why would a player do it?

Speaking only for myself, not having to do a blockchain transaction to sync ever again would be a huge value in terms of effort and sometimes stress about gas prices.

### But how would it work?

Here's one approach. Probably not the best, but it's stream of consciousness ....

1. Add "Buy monthly subscription" button to the game UI
2. When button is invoked, a minting transaction is sent to the block chain for a Monthly Player NFT with a timestamp in the metadata.
3. Add a "Refill supplies" button to the game UI
4. When a farm owner's account owns a non-expired Monthly Player NFT, pressing the "Refill supplies" button does the supply portion of what a Sync does today
5. (Optional) Add a "Buy yearly subscription" button to the game UI
6. (Optional) Update the Bumpkin UI to show how many monthly (and yearly) subscriptions the player has purchased (to show support and/or flex)

**Reply by RafaVLP** [reply]
Wouldn't it be risky to go a month without saving your progress on chain? Or by paying a monthly "subscription", would you be guaranteed to save your progress periodically?

**Reply by brian-man** [reply]
> Wouldn't it be risky to go a month without saving your progress on chain? Or by paying a monthly "subscription", would you be guaranteed to save your progress periodically?

The intent was for the resupply portion.

Saving your state to the chain could be done separately, and there's already some talk about decoupling the resupply and sync actions.  Related to on-chain metrics for DAU (daily active users) and such.

Sidenote: @adamhannigan Perhaps I'm misunderstanding the priorities, but I think it makes sense for "save to chain" to have no game fee if/when sync splits into two separate actions: sync and resupply.

**Reply by adamhannigan** [reply]
Thanks for raising this @brian-man 👏 

To be honest, we haven't yet explored a subscription model. The idea is interesting - we just need to do a lot more research into that area

**Reply by twilker** [reply]
I think this would break the game, as some NFTs like the toolshed are intended to increase the amount of seeds, which would be totally useless in this case. Theoretically, you could plant Sunflowers or Potatoes the whole day and mint far more SFL than you would normally. This would be an option which is helpful mostly for OGs which have maximum expansions. To be an attractive option you have to make is a bit cheaper and consider the gameplay changing aspects of it.

**Reply by brian-man** [reply]
If unlimited is hard to swallow...

Option B
$50 for 500 syncs with no or only 1 gas/transaction fee is also compelling to me

For the dev team: regular income, and most players would "overpay" because they wouldn't use all 500 every month
For the player: gas costs disappear, funding goes competely to the dev team

**Comment by gustavombueno** [comment]
I strongly support the idea and I think it should be the number 1 priority right now due to the high gas rates.

However, I did not like that they increased the price of the team fee from 0.10 to 0.15. This 50% increase seems like a government tax that took advantage of a situation to gain something in return.

The 0.10 fee is already very fair, and the team also receives % on withdrawals of SFLs and resources, in addition to % on transactions from OpeaSea to Nifty.

**Reply by adamhannigan** [reply]
Thanks for your feedback 🙏 The 0.15 fee is optional for players. We've kept the current 0.10 fee for those who want to continue with the structure that is in place

**Comment by pappajan69** [comment]
I run out of stone pickaxe and iron pickaxes and now  that treasury island is out will be shovels and drills so if we can get more of that and 1 sync give u 3x the seeds that will work for me

**Comment by GuilhermeSenaRJ** [comment]
I thought the suggestion was great, you can apply it as soon as possible! will it be an item within the game that in addition to replenishing items, it updates things within the blockchain? How would the goblin village look like?

**Comment by DeshFarm** [comment]
Acho super valido uma atualização, mesmo que momentânea em relação as taxas de gás...visto que está algo exorbitante os preços da rede polygon e muitos jogadores novos não conseguem recursos para as taxas se pagarem ou não tem muito dinheiro mesmo... jogadores mais antigo são sim afetados também mas em menor escala... eu acho muito valido essa proposta. 
Em relação aos dados frescos, é algo que mesmo que prejudique um pouco esse conhecimento de quantas pessoas estão ativas ou sincronizar as informações da fazenda com a rede, vai ser algo muito vantajoso para os jogadores passarem esse período o qual as taxas estão com preços altíssimos.

I think it's super worth an update, even if it's momentary in relation to gas fees...since the prices of the polygon network are somewhat exorbitant and many new players can't afford the fees if they pay or don't have a lot of money anyway... players older ones are also affected but on a smaller scale... I think this proposal is very valid.
Regarding fresh data, it is something that even if it affects a little this knowledge of how many people are active or synchronizing farm information with the network, it will be something very advantageous for players to pass this period in which rates are at very high prices.

**Comment by Ant-Redstone** [comment]
Sometimes I get a 1 dollar fee, then I simply plant wheat
Also, please conside that More Time playing without syncing means more transactions on sync, which means higher gas fees

Using treasure island as example:

Treasure island increases the amount of transactions aswell cuz we
Buy shovels (mint of shovel, burn of SFL, wood, stone)
Burn the shovels and Get a bunch of different items (mint and burn of em)
So we can consider that each different item will increase the sync fee by 6-7 transactions
and empty holes will increase by 5 transactions

in 20 shovels... that's a lot

So if u stay pushing 1.5 - 2.5 syncs per day with treasure island's low shovel/drill supply, it won't help even if this proposal get approved

**Comment by celpower** [comment]
Will I be able to buy in bulk (5 Bucks) and still sync normally saving the left over 4 Bucks for later when gas are high? It could also mean I have 4 Bucks saved but need to sync on chain to mint an item, so need to do a single sync and save the bucks for a later time.

**Comment by Ogrekin** [comment]
This is a great idea and very useful for players and I'm all up for providing more revenue for the team than feeding polygon...
The core team deserve and needs that income to keep going for right now they just lose more revenue as most players sync less..
I used to sync 2-6 times a day and now max one time a day ind I'm not alone in syncing less..
I see this as very good proposal and hope for it to be implemented, my only question is same as celpower if we can do normal syncs
in between bulk so it can be like we use first 2 bucks and then do a normal sync and then use remaining 3 bucks at a later stage?
If not I'm not sure it will be cheaper for us user as we by bulk and then be forced to use them when gas--fess are low..
Other than that it's very good proposal like it a lot...

**Comment by SarielWang93** [comment]
Please pay attention to the upper limit of hoarding, be sure to modify the upper limit of hoarding

**Comment by Yommka** [comment]
Hello, the polygon network for a comfortable game is no longer relevant, switch to another cheap network, for example, the raptoreum network there will cost a penny to synchronize and this network is faster, it also exists in web3

**Comment by Yommka** [comment]
Hello, the polygon network for a comfortable game is no longer relevant, switch to another cheap network, for example, the raptoreum network there will cost a penny to synchronize and this network is faster, it also exists in web3

Raptoreum (rtm)

**Reply by IvanSkazkin** [reply]
> Здравствуйте, полигональная сеть для комфортной игры уже не актуальна, переключитесь на другую дешевую сеть, например сеть раптореум там будет стоить копейки синхронизация и эта сеть быстрее, в веб3 она тоже есть
> 
> Раптореум (rtm)

Yeah, but what about the existing ones in the tax office? Do you know how difficult such a big project is supposed to be?

**Comment by Alaindrea** [comment]
I hate the idea of paying more to sync than if I just wait it out or suck up the fee. I feel like this goes against how you tried to keep the game accessible to people by only charging $5 to mint a farm. I think there needs to be a better way to track daily active users if tracking daily active users is that important. I don't have any idea how important it is or evven if it is important, so to be fair you should consider educating the community on Daily Active Users in general if you want informed opinions on this topic.

**Reply by IvanSkazkin** [reply]
_I don't understand your outrage. The command to relieve pressure, which has become expensive. That in a year there will be such problems with gas. So they solve the problem that has arisen._

**Reply by Ogrekin** [reply]
It is still a web3 game that needs transaction on chain and that fee (gas fee) is not anything sfl can do anything about...
Players still needs money to pay that gas fee so even if sfl is accessible for all, polygons gas fees might not be accessible for all and that is nothing sfl can do anything about it is still a blockchain game where polygon charges for a transaction on top of sync fee and it is that fee thats high now... sync fee+gas fee

**Comment by IvanSkazkin** [comment]
Adam, I noticed that you try to do everything according to lore. I wanted to offer to come up with a bust for the statue of the polygon (Sunflower Rock). After all, she broke the landfill at the same time. This upgrade you're planning is a great fit. Perhaps you could add a +1 store refresh for those who have this statue or - 5% on purchase to lock a dollar or something for your consideration. This would encourage OG and would be closely related to the game's lore. Thank you

You do not fall out, but vice versa. The price of the statue will increase by OS. This means that the percentage of sales will be higher. Therefore, this NFT bonus offer is a plus for everyone, for the team and for the community.

**Comment by caiofelipegq** [comment]
Maybe can use the new system to buy the 3 balloon tickets in a pack too? @adamhannigan

**Comment by TCoutz** [comment]
Are we still gonna be able to do the old sync if we want to? Or restock is going to be only trough block bucks? I want the option to make the direct sync to be always available.

I wonder if this proposal would have some impact on wishing well rewards. What do you guys think?

**Reply by IvanSkazkin** [reply]
As I understand it, you can buy 1 Block Bucks. This is the same as regular sync.

**Reply by TCoutz** [reply]
It's not the same. You pay for the block and for the fee. This could lead to a situation where gas fees where low but you had to buy one block just to restock and end up paying more than if you just synced without the block.

**Comment by franvi** [comment]
Syncing should be separated from restocking
Move restocking on server but mandatory syncing at least once in every two weeks or once a month as the case maybe.

Get bucks any amount you intend to buy pay a one time gas fee which essentially means you can just restock off-chain you only need to do the mandatory syncing but that will be without cost just pay gas for it definetly you will have to sync if you are minting an item will buck sync move your data on-chain.

And what about expansion

**Comment by eliasSFL** [comment]
I just realised, there needs to be an adjustment the limit for SFL gains and egg gains, otherwise the feature will be useless cuz the game will be asking you to sync

**Reply by Sunflowerfarmer** [reply]
> _The following is in draft mode - it is an early idea but we wanted to gather community feedback since this is a pressing concern. We will share this in announcements once details are finalised_
> 
> The purpose of this proposal is to discuss the issue of high gas fees that have been occurring across Polygon the last 5 days. Recently these fees have been 5x due to a range of new projects on the Blockchain.
> 
> Unfortunately this has led to players unable to sync, expensive fees and/or failed transactions.
> 
> The team are investigating an approach that will ensure gameplay is not drastically affected during these periods of high traffic on the Blockchain.
> 
> **The balance of Web2 and Web3**
> 
> The Sunflower Land team is deeply passionate about on-chain gameplay, evident in our prototype which was 100% on the Blockchain. Since then we have learnt it is best to perform some actions on a game server and reserve others for Blockchain. This reduces transaction fees and improves the games UX significantly without the constant need to wait for transactions.
> 
> There is a fine balance that we strive for when integrating Web2 and Web3 elements. Our goal continues to maximise on chain activity whilst ensuring the game is playable and user friendly.
> 
> ## Block Bucks
> We are interested in launching the idea of "Block Bucks" - name is to be decided, please provide suggestions 🙏
> 
> 1 Block Buck could be used to **instantly** restock your shops, instead of 'Syncing your data on chain'. This means players would not need to wait for a transaction and pay the associated gas fees.
> 
> **How do you get Block Bucks?**
> 
> You would be able to purchase 5 Block Bucks within the game for **$X**. This would trigger a transaction that would:
> 
> 1. Sync your data on Chain
> 2. Add **X** Block Buck tickets to your inventory.
> 
> Once the transaction completes, you can then use these tickets to restock your inventory.
> 
> The approach above enables a player to restock their shop 5 times whilst only paying the gas fees on 1 transaction. In essence, a player pays roughly 1/5th of the gas fees.
> 
> **Pros**
> 
> * Players spend less time waiting for transactions / restocks are instant
> * Players spend less money on gas fees
> 
> **Cons**
> 
> * On chain daily active users would decrease
> * On chain data is available later than normal (only once a user gets Block Bucks)
> * Input price for resources decreases. If players are paying less gas fees, MATIC input is less for output of resources
> 
> On a side note, the team will be investigating a Daily Rewards on chain transaction that can use insignificant gas (<0.02) - this will become the main driver for capturing DAU (Daily Active Users)
> 
> ## Pricing Structure
> We want to encourage users to sync on chain more often to ensure we remain that balance between web2 and web3 gaming. The pricing structure we propose takes an inverse structure to buying in bulk. There are multiple options to choose which enables players to adapt to current blockchain conditions.
> 
> The following calculations assume:
> 
> * Average gas prices for a sync are around $0.11-0.13
> * Spiked gas prices for a sync current are around $0.30-$0.35
> 
> 1 Buck = $0.10 (current price in game) 5 Bucks = $0.75 Total = $0.15 each
> 
> When a Block Buck is purchased, the standard gas fees would be applied while syncing on chain. Below we look at the overall transaction price when purchasing the following with average gas fees of **$0.12**
> 
> 1 Buck - $0.10 + $0.12 = $0.22 per sync (average price players were paying) 5 Bucks -($0.75 + $0.12) / 5 = $0.174 per sync
> 
> In the above scenario of average Blockchain conditions, a player who chose 5 Bucks would save roughly $0.05 per sync and an **overall savings of $0.47**
> 
> Below we look at the overall transaction price when purchasing the following with average gas fees of **$0.35**
> 
> 1 Buck - $0.10 + $0.35 = $0.45 per sync (current price players are paying with spiked fees) 5 Bucks -($0.75 + $0.35) / 5 = $0.22 per sync
> 
> In the above **spiked** scenario, players who chose 5 Bucks would save roughly $0.23 per sync and an **overall savings of $1.40**
> 
> ## Timeline
> We estimate this feature would take roughly 2-3 days to roll out and would not have a significant impact on our roadmap items 🚀
> 
> ### Summary
> Overall, the proposed Blocks solution saves money for players on gas fees in all Blockchain traffic conditions & ensures smoother of gameplay without waiting for long-running transactions.
> 
> However, we must tread carefully with this approach as it means Blockchain data becomes less 'fresh' and our perceived on chain users would decrease.
> 
> As always we appreciate all kinds of feedback and suggestions ❤️

New to GitHub idk how it works...

Development runs on 0.1$, per sync. 
I suggest 30 sync at once, which costs 6$ (2x normal sync, bcoz it allows players to sync less if they target specific crops. But again players saves 3$ for gas itself so it's same for players in some way) 
In 30 sync all seeds and tools will be available in 30x quantity. Buying cap will still be same (can't buy and hoard more than 400 sf seeds) 
This will keep all data on server side. Users need to sync again if they want something on chain. 
It's win win.

**Comment by cryptokohai** [comment]
How will block bucks interact with non-restock features. I use the example of FB - requiring "sync" every "X" number of wood harvested, will a "sync" still be required to reset the "X" number of wood harvested or will a "restock" suffice? There are many other features that currently relies on a "sync" to reset, such as eggs, crops, etc.

**Comment by future2back** [comment]
Success! Suggest a confirmation message. #2220

**Reply by future2back** [reply]
Solved on #2221

**Comment by Xeko-92** [comment]
I would like a confirmation window when using the block bucks.

**Reply by IvanSkazkin** [reply]
Agree

**Reply by future2back** [reply]
Solved on #2221

**Comment by buenti** [comment]
Adam, devs, please increase the hoard limit, is so low in eggs that the 5x bucks almost doesn't have sense in eggs. With the coop and the 2 hen the limit is so short

**Comment by IvanSkazkin** [comment]
Adam, I see a problem with your system after using it.

I bought 5 Block Bucks. I have a green amulet. I planted sunflowers. After 3 store updates, I'm going to do a sync that collects sunflowers.

I can't buy Block Bucks because I have 2 more in stock. But beds also happen, I can’t, there is no shovel (it doesn’t work) and the game calls me to do dependent synchronization. And synchronization costs $1.

It turns out the idea that it is to reduce our spending, and I only spend more, because there is such a synchronization. I am not receiving store updates or Block Bucks. It turns out empty synchronization.

**Comment by Myth727** [comment]
I used a blockblocks to restock and very shortly after had to sync due to hoarding limits on wood, now it wants me to sync again for those hoarding limits and gas fees show almost 1 matic. I'm sure yall will figure something out, just bringing it up :)

**Reply by ExtremeCheddar** [reply]
Would be great to have the hoarding sync removed completely as we now have to sync every time we buy block bucks

**Reply by IvanSkazkin** [reply]
I have such a problem with sunflowers, not with their beds, they are removed.

**Comment by Xeko-92** [comment]
I think it would be nice if the buy 1 Block Buck option was available as long as you had less than 5 block bucks. So you can sync on chain and still get a block buck if let's say you bought 5 before, used 3, but now need to sync to avoid the hoarding limit.

**Reply by IvanSkazkin** [reply]
It sounds good!

**Comment by IvanSkazkin** [comment]
Please choose stocking for vegetables or add a shovel. I have 3 blocks of bucks now. I planted sunflowers. I can't collect them. The beds are busy and writes that I should synchronize. Synchronization costs 1 matic. And it will be an empty synchronization, because I will not receive tickets. I can't buy them until I spend the rest

**Comment by IvanSkazkin** [comment]
I logically remove the resource limit after each use of a block of bucks.
But if you want not to lose because of this income.
Please make tickets to replenish the accumulation.
For example, a x10 overstock upgrade costs (at your discretion).

Adding another hoarder ticket sounds interesting

**Comment by IvanSkazkin** [comment]
Adam thought you could add a block of bucks to your payment network. Leave the polygon, but add other networks. This would help to get off the big gas of different networks. While one network is loaded, they buy less loaded
